wagtailio/static/sass/components/_code-snippet.scss

```diff
@@ -0,0 +1,82 @@
+.code-snippet {
+    $root: &;
+    background-color: $color--indigo;
+    color: $color--white;
+    display: flex;
```

Comment:
Note: In this case, do we actually want to show this to the user? In other places, it makes sense if there are technical articles but I'm unsure of the value we're providing to mobile users.

Reply:
I think we discussed with Ben and said it still provides info to a mobile user, even if they can't use the command on their phone they can make a mental note of it for later.